common/persistence/nosql/nosqlplugin/cassandra/configStore.go

```diff
@@ -58,7 +56,7 @@ func (db *cdb) SelectLatestConfig(ctx context.Context, rowType int) (*persistenc
 
 	query := db.session.Query(templateSelectLatestConfig, rowType).WithContext(ctx)
 	err := query.Scan(&rowType, &version, &timestamp, &data, &encoding)
-	if err == gocql.ErrNotFound {
+	if db.IsNotFoundError(err) {
```

Comment:
I think we should move the logic to make it work for all nosqlplugins

Reply:
happy to do that, but I don't quite understand